angular/src/app/user-area/qr-code-dialog/qr-code-dialog.component.ts

```diff
@@ -25,7 +26,8 @@ export class QrCodeDialogComponent implements OnInit {
   private loadQrCode(): void {
     this.userAreaService.pairing().subscribe(
       (res: any) => {
-        this.qrcode = res.body;
+        this.qrcode = JSON.parse(res.body)['base64QrCode'];
```

Comment:
![image](https://user-images.githubusercontent.com/43032659/59198777-d4048f00-8b94-11e9-976e-8325599059e2.png)

This code generates an error the first time you try to configure 2FA.
Furthermore, It would be better to declare an interface and use it instead of ['something'], for example:

export interface Response {
    body: {
        base64QrCode?: any;
        secret?: any;
    }
}

private loadQrCode(): void {
// Using a pipe to parse
    this.userAreaService.pairing().pipe(res => JSON.parse(res)).subscribe(
        // Using interface
        (res: Response) => {
            this.qrcode = res.body.base64QrCode;
        }
    )
}

The interface could be inside user-area/models/response.model.ts